main.py

```diff
@@ -83,8 +83,17 @@ def run(starter):
                         help='Specify the mesh file path to chop. This will override the mesh file in the config yaml')
     args = parser.parse_args()
 
-    # load specified or default config file
-    config = Configuration(args.config)
+    # print help if no arguments specified
+    if len(sys.argv) < 1:
```

Comment:
I believe this is impossible because sys.argv[0] is the file name so if this script is being run then len(sys.argv) is at least 1. Also, if no arguments are specified, don't we want the defaults to work?

Reply:
We could check if there are no arguments specified, and if this is the case tell the user what defaults are being used

Reply:
Good catch on `sys.argv[0]`! See [71e40ec](https://github.com/gregstarr/pychop3d/pull/28/commits/71e40eca235c1aa00fba83fedfa42e2f551b7599) with a quick fix.

Reply:
Config/mesh information should probably be printed regardless of arguments. Maybe adding logging would be a good solution? The first couple messages could be something like...

```
2020-03-13 21:54:40,758  pychop3d [INFO]  Using mesh: shoe rack v13.stl
2020-03-13 21:55:42,483  pychop3d [INFO]  Using config: shoerack_config.yml
```